tests/steps/__init__.py

```diff
@@ -1,9 +1,10 @@
-from jwst.stpipe import (Pipeline, Step)
-from jwst import datamodels
-from jwst.datamodels import (
-    ImageModel,
-    ModelContainer,
-)
+from stpipe import (Pipeline, Step)
```

Comment:
I really hate how we have everything in `__init__.py` in this test dir.  Would be nice to refactor this in the future to be more standard.

Reply:
I agree, this needs cleanup.